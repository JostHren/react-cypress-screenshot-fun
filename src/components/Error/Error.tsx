import { Container, Stack, Typography } from '@mui/material';

interface ErrorPageProps {
  title: string;
  message: string;
}

export const ErrorPage = ({ title, message }: ErrorPageProps) => {
  return (
    <Container>
      <Stack spacing={5}>
        <Typography variant="h1">{title}</Typography>
        <Typography variant="h2">{message}</Typography>
      </Stack>
    </Container>
  );
};
