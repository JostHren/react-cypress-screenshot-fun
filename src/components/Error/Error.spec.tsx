import { render } from '@testing-library/react';
import { ErrorPage } from './Error';

describe('Error component', () => {
  it('CCChat component to render', () => {
    const { container } = render(<ErrorPage title="mock-title" message="mock-message" />);

    expect(container).toMatchSnapshot();
  });
});
