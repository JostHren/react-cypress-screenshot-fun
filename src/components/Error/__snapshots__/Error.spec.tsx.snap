// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error component > CCChat component to render 1`] = `
<div>
  <div
    data-component-name="Container"
  >
    --- PROPS
    {}
    --- CHILDREN
    <div
      data-component-name="Stack"
    >
      --- PROPS
      { spacing: 5 }
      --- CHILDREN
      <div
        data-component-name="Typography"
      >
        --- PROPS
        { variant: 'h1' }
        --- CHILDREN
        mock-title
      </div>
      <div
        data-component-name="Typography"
      >
        --- PROPS
        { variant: 'h2' }
        --- CHILDREN
        mock-message
      </div>
    </div>
  </div>
</div>
`;
